### DISCUSSION TITLE
Tagaction IgnoreIncapacitatedCharacters does not work on stunned characters

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14744] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In Tagaction,  `IgnoreIncapacitatedCharacters="true"` doesnt seem to properly ignore incapacitated characters
https://github.com/FakeFishGames/Barotrauma/blob/0e8fb6569d2810e2f8ad5fb17b4bba546cc5739a/Barotrauma/BarotraumaShared/SharedSource/Events/EventActions/TagAction.cs#L28

At the same time, it defaults to "true", so if the bug is fixed it might also possibly break events 

This was discovered when trying to tag _non-dead_ NPCs and it feels more relevant to have a `ignoredeadcharacters` than ignoreincapacitated but at this point I don't even know

Test event:    
```xml
    <ScriptedEvent identifier="incapacitatedTEST">
      <TagAction criteria="player" tag="notincapacitated" IgnoreIncapacitatedCharacters="true" ContinueIfNoTargetsFound="true" />
      <StatusEffectAction targettag="notincapacitated" >
        <StatusEffect target="this" >
          <SpawnItem identifier="handcuffs" spawnposition="ThisInventory" Equip="true" /> 
        </StatusEffect>
      </StatusEffectAction>
    </ScriptedEvent>
```

### Reproduction steps

giveaffliction stun 10
triggerevent incapacitatedTEST

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
stunned character is not considered incapacitated

**Comment by A2ur3** [comment]
```cs
public bool IsIncapacitated
{
    get
    {
        if (IsUnconscious) { return true; }
        return CharacterHealth.IsParalyzed;
    }
}
```

**Comment by Regalis11** [comment]
Doesn't seem to be bugged, stunned characters are just not considered "incapacitated". Even though I agree that's somewhat unintuitive, stuns are so common and often not completely "debilitating" that I don't think we should consider it to be "incapacitation".

**Reply by ubertpendragon** [reply]
Seems like it doesnt work on dead characters either
